components/toggle/theme.css

```diff
@@ -0,0 +1,363 @@
+@import 'variables.css';
+@import '@teamleader/ui-colors/index.css';
+
+:root {
+  --toggle-track-height-small: 18px;
+  --toggle-track-width-small: 30px;
+  --toggle-thumb-width-small: 12px;
+
+  --toggle-track-height-medium: 24px;
+  --toggle-track-width-medium: 42px;
+  --toggle-thumb-width-medium: 18px;
+
+  --toggle-track-height-large: 30px;
+  --toggle-track-width-large: 54px;
+  --toggle-thumb-width-large: 24px;
+
+  --toggle-track-padding: 3px;
+}
+
+.toggle {
```

Comment:
Can we not rewrite this using parent selectors? Or do we need another PostCSS plugin for that 🤔 

```css
.thumb {
  .medium & {
    /* whatever */
}
```

Reply:
Yes, it works with `postcss-nested`. Going to refactor my css tomorrow.

Reply:
Tried to refactor with using parent selectors but it didn't really work out. A lot of things weren't possible or were giving me css specificity issues.

 :ok: by @lowiebenoot on 19/10/2017 11:51:53